src/main/java/co/norse/chat/controller/UserController.java

```diff
@@ -0,0 +1,61 @@
+package co.norse.chat.controller;
+
+import co.norse.chat.User;
+import co.norse.chat.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class UserController {
+    @Autowired
+    UserService userService;
+
+    @GetMapping(value = "/users")
+    public ResponseEntity<List<User>> getAllUsers() {
+        final List<User> users = userService.findAllUsers();
+        return users != null && !users.isEmpty()
+                ? new ResponseEntity<>(users, HttpStatus.OK)
+                : new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+
+    @PostMapping(value = "/users")
+    public ResponseEntity<?> addUser(@RequestBody User user) {
```

Comment:
pls rework to
`    @PostMapping(value = "/users")
    @ResponseStatus(HttpStatus.CREATED)
    public void addUser(@RequestBody User user) {
        userService.addUser(user);
    }`